### PR TITLE
Add deploy-test-data script

### PR DESCRIPTION
**Pull request by matthieu-foucault**
- moved all row inserts in the `schema/data` folder
- added `schema/.cas-ggircs` submodule to load SWRS schema
- added a dummy swrs xml report
- added `schema/deploy-test-data.sh` script
- added `install_asdf_tools` make target and build postgres with libxml support
- renamed `watch-schema-sqitch` config to `watch-schema-data` to reflect new behavior (runs the new script instead of sqitch)
- various fixes in `get_swrs*` functions

### CODE REVIEW

File: schema/deploy/function_get_swrs_operator_contact_data.sql [hunk 1]
```diff
@@ -54,12 +54,16 @@ begin;
...
+          cast(
+            concat(
+              'unit ' || _contact_add.mailing_address_unit_number || ', ',
+              _contact_add.mailing_address_street_number || ' ',
+              _contact_add.mailing_address_street_number_suffix || ' ',
+              _contact_add.mailing_address_street_name || ' ',
+              _contact_add.mailing_address_street_type || ' ',
+              _contact_add.mailing_address_street_direction
+            )
+          as varchar(1000)),
```
Comment (matthieu-foucault) [inline schema/deploy/function_get_swrs_operator_contact_data.sql]: @hamzajaved This allows for fields to be `null`, while not printing "unit" if there is no unit number.
If one of the strings is null `||` outputs `null`, while `concat` ignores `null` parameters

Comment (wenzowski) [reply schema/deploy/function_get_swrs_operator_contact_data.sql]: It saddens me that we have db testing set up in [cas-ggircs](https://github.com/bcgov/cas-ggircs/tree/develop/test) but not here. This is exactly the sort of comment that should be described by a unit test.

**Comment by wenzowski** [comment]
Still waiting on [circle outage](https://status.circleci.com/incidents/4wbtp4jdd3c0) but this looks fab at a quick skim :)

**Comment by matthieu-foucault** [comment]
@wenzowski CircleCI has finally caught up with its backlog :smile:

File: schema/data/portal/form_result.sql [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  (13,5,2,2, '2019-09-17 14:49:54.191757-07', '{"moduleThroughputAndProductionData": [{"units": ",m3", "comments": "Mollitia ex non temp", "quantity": 1100, "processingUnit": "Sales Compression", "associatedEmissions": "Molestiae blanditiis", "attributableFuelPercentage": 52}]}'),
+  (14,1,2,2, '2019-09-17 14:49:54.191757-07', '{"operationalRepresentativeInformation": [{"fax": "+1 (403) 112-7963", "phone": "+1 (475) 164-3308", "position": "Culpa officiis liber", "lastName": "Velit facilis offici", "firstName": "Facere nihil minima ", "emailAddress": "mojoqub@mailinator.com", "mailingAddressCity": "Nesciunt et tenetur", "mailingAddress": "Amet optio impedit", "mailingAddressProvince": "British Columbia", "mailingAddressPostalCode": "h0h0h0"}]}')
```
Comment (matthieu-foucault) [inline schema/data/portal/form_result.sql]: This data uses an older format for the results and needs to be edited

**Pull request merged**